diff --git a/lib/commonjs/components/bottomSheet/BottomSheet.js b/lib/commonjs/components/bottomSheet/BottomSheet.js
index 40725c071f23498d5a3053c0b307ab3f17566e18..46a512c0daa5e0e7c5d6e47c58a1f4eb96d3747d 100644
--- a/lib/commonjs/components/bottomSheet/BottomSheet.js
+++ b/lib/commonjs/components/bottomSheet/BottomSheet.js
@@ -1251,7 +1251,7 @@ const BottomSheetComponent = /*#__PURE__*/(0, _react.forwardRef)(function Bottom
         children: [BackdropComponent ? /*#__PURE__*/(0, _jsxRuntime.jsx)(BackdropComponent, {
           animatedIndex: animatedIndex,
           animatedPosition: animatedPosition,
-          style: _reactNative.StyleSheet.absoluteFillObject
+          style: _reactNative.StyleSheet.absoluteFill
         }) : null, /*#__PURE__*/(0, _jsxRuntime.jsx)(_bottomSheetHostingContainer.BottomSheetHostingContainer, {
           shouldCalculateHeight: !$modal,
           layoutState: animatedLayoutState,
diff --git a/lib/commonjs/components/bottomSheetBackdrop/styles.js b/lib/commonjs/components/bottomSheetBackdrop/styles.js
index c1f62f66ab1932bd06a2937871f21db130a6d82a..55ec957381c1c47a9b42e499a64d7dda37c9f570 100644
--- a/lib/commonjs/components/bottomSheetBackdrop/styles.js
+++ b/lib/commonjs/components/bottomSheetBackdrop/styles.js
@@ -7,7 +7,7 @@ exports.styles = void 0;
 var _reactNative = require("react-native");
 const styles = exports.styles = _reactNative.StyleSheet.create({
   backdrop: {
-    ..._reactNative.StyleSheet.absoluteFillObject,
+    ..._reactNative.StyleSheet.absoluteFill,
     backgroundColor: 'black'
   }
 });
diff --git a/lib/commonjs/components/bottomSheetBackground/styles.js b/lib/commonjs/components/bottomSheetBackground/styles.js
index 4e4fb38373692772256f0f8fd52417d76d17aba5..d5aab4d973be3ceb71f2ea17a6c2874c6b5c1f61 100644
--- a/lib/commonjs/components/bottomSheetBackground/styles.js
+++ b/lib/commonjs/components/bottomSheetBackground/styles.js
@@ -6,7 +6,7 @@ Object.defineProperty(exports, "__esModule", {
 exports.styles = void 0;
 var _reactNative = require("react-native");
 const styles = exports.styles = _reactNative.StyleSheet.create({
-  container: _reactNative.StyleSheet.absoluteFillObject,
+  container: _reactNative.StyleSheet.absoluteFill,
   background: {
     backgroundColor: 'white',
     borderRadius: 15
diff --git a/lib/commonjs/components/bottomSheetHostingContainer/styles.js b/lib/commonjs/components/bottomSheetHostingContainer/styles.js
index c563e174c156a9b653a25224954b1d7a8e88418c..e2e5c5c3cc5e699d76c406483938d8000a1030d7 100644
--- a/lib/commonjs/components/bottomSheetHostingContainer/styles.js
+++ b/lib/commonjs/components/bottomSheetHostingContainer/styles.js
@@ -7,7 +7,7 @@ exports.styles = void 0;
 var _reactNative = require("react-native");
 const styles = exports.styles = _reactNative.StyleSheet.create({
   container: {
-    ..._reactNative.StyleSheet.absoluteFillObject,
+    ..._reactNative.StyleSheet.absoluteFill,
     pointerEvents: 'box-none'
   }
 });
diff --git a/lib/module/components/bottomSheet/BottomSheet.js b/lib/module/components/bottomSheet/BottomSheet.js
index 561ad524ac6ed22e9e2284097b964923751ee9aa..a75453ea846c530a121dc344beb83d2a4c52c77d 100644
--- a/lib/module/components/bottomSheet/BottomSheet.js
+++ b/lib/module/components/bottomSheet/BottomSheet.js
@@ -1244,7 +1244,7 @@ const BottomSheetComponent = /*#__PURE__*/forwardRef(function BottomSheet(props,
         children: [BackdropComponent ? /*#__PURE__*/_jsx(BackdropComponent, {
           animatedIndex: animatedIndex,
           animatedPosition: animatedPosition,
-          style: StyleSheet.absoluteFillObject
+          style: StyleSheet.absoluteFill
         }) : null, /*#__PURE__*/_jsx(BottomSheetHostingContainer, {
           shouldCalculateHeight: !$modal,
           layoutState: animatedLayoutState,
diff --git a/lib/module/components/bottomSheetBackdrop/styles.js b/lib/module/components/bottomSheetBackdrop/styles.js
index 4bf1c1b2eaacac963c4601fe6f5eed7d324f4cd2..d0cbc8f72592c3ac5f7e006f8ba4b639bb17f99e 100644
--- a/lib/module/components/bottomSheetBackdrop/styles.js
+++ b/lib/module/components/bottomSheetBackdrop/styles.js
@@ -3,7 +3,7 @@
 import { StyleSheet } from 'react-native';
 export const styles = StyleSheet.create({
   backdrop: {
-    ...StyleSheet.absoluteFillObject,
+    ...StyleSheet.absoluteFill,
     backgroundColor: 'black'
   }
 });
diff --git a/lib/module/components/bottomSheetBackground/styles.js b/lib/module/components/bottomSheetBackground/styles.js
index 863e5d6328fb097259de67a10cdccf8225f697ca..9802fa21d6e9d4d4262ee69d913320656a180d3d 100644
--- a/lib/module/components/bottomSheetBackground/styles.js
+++ b/lib/module/components/bottomSheetBackground/styles.js
@@ -2,7 +2,7 @@
 
 import { StyleSheet } from 'react-native';
 export const styles = StyleSheet.create({
-  container: StyleSheet.absoluteFillObject,
+  container: StyleSheet.absoluteFill,
   background: {
     backgroundColor: 'white',
     borderRadius: 15
diff --git a/lib/module/components/bottomSheetHostingContainer/styles.js b/lib/module/components/bottomSheetHostingContainer/styles.js
index 49412c39fae418bc07d457ea29005c150ee0d94d..24b8d0077829846995ab3c85cfc3421470c60e25 100644
--- a/lib/module/components/bottomSheetHostingContainer/styles.js
+++ b/lib/module/components/bottomSheetHostingContainer/styles.js
@@ -3,7 +3,7 @@
 import { StyleSheet } from 'react-native';
 export const styles = StyleSheet.create({
   container: {
-    ...StyleSheet.absoluteFillObject,
+    ...StyleSheet.absoluteFill,
     pointerEvents: 'box-none'
   }
 });
diff --git a/src/components/bottomSheet/BottomSheet.tsx b/src/components/bottomSheet/BottomSheet.tsx
index 6748c93cac1a29c745480fe2282b8d8f6b36226f..2bc3f7f8fc26f7b59445a6af3768b9fcd14063ff 100644
--- a/src/components/bottomSheet/BottomSheet.tsx
+++ b/src/components/bottomSheet/BottomSheet.tsx
@@ -1785,7 +1785,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
               <BackdropComponent
                 animatedIndex={animatedIndex}
                 animatedPosition={animatedPosition}
-                style={StyleSheet.absoluteFillObject}
+                style={StyleSheet.absoluteFill}
               />
             ) : null}
             <BottomSheetHostingContainer
diff --git a/src/components/bottomSheetBackdrop/styles.ts b/src/components/bottomSheetBackdrop/styles.ts
index 6151fe8920af469884c2bfad20594b54f17e7b99..973796c8d2c23c7b3df8afe5be30cdb9d233ce88 100644
--- a/src/components/bottomSheetBackdrop/styles.ts
+++ b/src/components/bottomSheetBackdrop/styles.ts
@@ -2,7 +2,7 @@ import { StyleSheet } from 'react-native';
 
 export const styles = StyleSheet.create({
   backdrop: {
-    ...StyleSheet.absoluteFillObject,
+    ...StyleSheet.absoluteFill,
     backgroundColor: 'black',
   },
 });
diff --git a/src/components/bottomSheetBackground/styles.ts b/src/components/bottomSheetBackground/styles.ts
index 45d8dfaa3c25bf9b2e9bbd8066b9024109d7b4f7..5c4da0a61f2d176ffbaf0d820fcd00377cdd6d37 100644
--- a/src/components/bottomSheetBackground/styles.ts
+++ b/src/components/bottomSheetBackground/styles.ts
@@ -1,7 +1,7 @@
 import { StyleSheet } from 'react-native';
 
 export const styles = StyleSheet.create({
-  container: StyleSheet.absoluteFillObject,
+  container: StyleSheet.absoluteFill,
   background: {
     backgroundColor: 'white',
     borderRadius: 15,
diff --git a/src/components/bottomSheetHostingContainer/styles.ts b/src/components/bottomSheetHostingContainer/styles.ts
index 95826d9ccd46f05432d38414306086d66a33d12f..830dda41005f37049a9e352da07fee6449602b72 100644
--- a/src/components/bottomSheetHostingContainer/styles.ts
+++ b/src/components/bottomSheetHostingContainer/styles.ts
@@ -1,5 +1,5 @@
 import { StyleSheet } from 'react-native';
 
 export const styles = StyleSheet.create({
-  container: { ...StyleSheet.absoluteFillObject, pointerEvents: 'box-none' },
+  container: { ...StyleSheet.absoluteFill, pointerEvents: 'box-none' },
 });
